activation/poetdb.go

```diff
@@ -60,7 +60,9 @@ func (db *PoetDb) ValidateAndStoreMsg(data []byte) error {
 
 // Validate validates a new PoET proof.
 func (db *PoetDb) Validate(proof types.PoetProof, poetID []byte, roundID string, signature []byte) error {
-
+	if len(poetID) < 5 {
```

Comment:
what is 5? maybe define a constant to explain?

Reply:
the 5 was used below for short id, and it was crashing if id was empty

Reply:
i was thinking changing it to a named constant makes the code more readable. up to you.